examples/jsm/nodes/accessors/LineMaterialNode.js

```diff
@@ -0,0 +1,31 @@
+import MaterialNode from './MaterialNode.js';
+import { addNodeClass } from '../core/Node.js';
+import { nodeImmutable } from '../shadernode/ShaderNode.js';
+
+class LineMaterialNode extends MaterialNode {
+
+	constructor( scope ) {
```

Comment:
Do we need such a pass-through constructor? I think these lines (7-11) can be just removed.